examples/native/native-server.rs

```diff
@@ -9,7 +9,9 @@ mod native {
     };
 
     pub(crate) fn main_native() {
-        env_logger::init();
+        env_logger::builder()
```

Comment:
A drive-by: while debugging on native I found the logging timestamps were missing millisecond detail.